src/tests/GC/API/GC/GetConfigurationVariables.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+/* 
+ * Tests ReadConfigurationVariables() 
+ */
+
+using System;
+public class Test_GetConfigurationVariables
+{
+    public static int Main()
```

Comment:
Instead of writing a `Main` use the xUnit `Fact` and a proper `Main` will be autogenerated for you.

For example,

```csharp
[Fact]
public void CollectAllVariables()
{
    var configurations = GC.GetConfigurationVariables();
    Assert.True(configurations.Count != 0);
}
```

Reply:
I'd like to see code that enumerates through all the strings and makes sure none are `null` or `string.Empty`. The values should also be confirmed to not contain a `null` - I assume that is true.